### ISSUE TITLE
[BUG] Dark mode - Active nav tab is fully white (unreadable)

### ISSUE THREAD

**Issue by SuperITMan** [compodoc/compodoc#1492]
##### **Overview of the issue**

In versions > 1.1.19, the active tab has background-color white **and** font color white. That causes the nav tab to be unreadable. 

In compodoc <=1.1.19:
![image](https://github.com/compodoc/compodoc/assets/10333565/d9c7f836-dd65-4dda-8791-f0043a1421cf)

In compodoc > 1.1.19
![image](https://github.com/compodoc/compodoc/assets/10333565/eb8ae032-5b10-4291-a5a2-5a14419ec673)

##### **Operating System, Node.js, npm, compodoc version(s)**

- NodeJS: 20
- npm: 10
- compodoc: >1.1.19

##### **Angular configuration, a `package.json` file in the root folder**

Angular version 16.

##### **Compodoc installed globally or locally ?**

locally

##### **If possible sourcecode of the file where it breaks**

The issue is that the `active` class is now applied on the `<a>` tag instead of the `<li>` tag as it was before.
https://github.com/compodoc/compodoc/blob/a7162ebdab5f217e86b6c047b67ed14fe82d85ef/src/resources/styles/material.css#L123-L125

HTML code in `compodoc@1.1.19`:
```html
<ul class="nav nav-tabs" role="tablist">
        <li class="active">
            <a href="#info" role="tab" id="info-tab" data-toggle="tab" data-link="info">Info</a>
        </li>
        <li>
            <a href="#source" role="tab" id="source-tab" data-toggle="tab" data-link="source">Source</a>
        </li>
        <li>
            <a href="#templateData" role="tab" id="templateData-tab" data-toggle="tab" data-link="template">Template</a>
        </li>
        <li>
            <a href="#tree" role="tab" id="tree-tab" data-toggle="tab" data-link="dom-tree">DOM Tree</a>
        </li>
</ul>
```

HTML code in `compodoc@1.1.20`:
```html
<ul class="nav nav-tabs" role="tablist">
        <li class="nav-item">
            <a href="#info" class="nav-link active" role="tab" id="info-tab" data-bs-toggle="tab" data-link="info" aria-selected="true">Info</a>
        </li>
        <li class="nav-item">
            <a href="#source" class="nav-link" role="tab" id="source-tab" data-bs-toggle="tab" data-link="source" aria-selected="false">Source</a>
        </li>
        <li class="nav-item">
            <a href="#templateData" class="nav-link" role="tab" id="templateData-tab" data-bs-toggle="tab" data-link="template">Template</a>
        </li>
        <li class="nav-item">
            <a href="#tree" class="nav-link" role="tab" id="tree-tab" data-bs-toggle="tab" data-link="dom-tree">DOM Tree</a>
        </li>
</ul>
```

##### **Motivation for or Use Case**

Fix dark mode.

**Comment by AshishK9677** [comment]
Hello, I am new here, Can I also start looking into this issue ?

**Comment by vogloblinsky** [comment]
@SuperITMan coudl you do the test in https://compodoc.github.io/compodoc-demo-todomvc-angular/components/EmptyComponent.html#info
Seems to be fixed

**Comment by vogloblinsky** [comment]
Cannot reproduce, @SuperITMan reopen if not fixed with last version, 1.1.25 today.
